backend/src/main/resources/application.yaml

```diff
@@ -47,8 +47,9 @@ server:
 #    key-store-type: pkcs12
 #    key-alias: localhost
 #    key-password: password
-#logging:
-#  level:
+logging:
+  level:
 #    KC_WEB_CLIENT: DEBUG
+    LOAD_TEST: DEBUG
```

Comment:
Don't forget to turn this off.

Reply:
Addressed by https://github.com/bcgov/moh-keycloak-user-management/pull/61/commits/72b2a37b2520df71ff527fcfdd4341c46084eac5